### PR TITLE
Upgrade to Varnish 4.1.5, Varnish modules 0.11.0 plus add vmod_digest

### PR DESCRIPTION
**Pull request by petercv**

### CODE REVIEW

File: varnish/Dockerfile [hunk 1]
```diff
@@ -6,5 +6,4 @@ RUN /tmp/docker-install.sh \
  && rm -rvf /tmp/*
 
 EXPOSE 6081
-ENTRYPOINT ["/usr/local/bin/chaperone"]
-CMD ["--user", "varnish"]
+CMD ["/usr/local/bin/chaperone", "--user", "varnish"]
```
Comment (avoinea) [inline varnish/Dockerfile:9]: Any good reason you've removed the entrypoint?

**Comment by avoinea** [comment]
@petercv Thank you! 

See my comments on code ;)

File: varnish/Dockerfile (hunk 1 above)
Comment (petercv) [reply varnish/Dockerfile:9]: Yes, the entrypoint is always executed even if you want to start the container with a different command (for example "sh" or "bash"). Moving chaperone to the CMD allows you to more easily start the container with a totally different command (for whatever reason).

Comment (avoinea) [reply varnish/Dockerfile:9]: @petercv I see, thanks!

**Pull request merged**